src/components/LanguageSelector.vue

```diff
@@ -40,6 +42,10 @@ const languageDropdownOptions = computed(() => {
     };
   });
 });
+
+function onLanguageChange(event) {
+  sessionStorage.setItem(`${isLevante ? 'levante' : 'roar'}PlatformLocale`, event.value);
```

Comment:
Why does this need to be different for ROAR and LEVANTE? Could we just set a `platformLocal` value?